atlas-sdk/src/main/java/com/atlas/sdk/AtlasSdk.kt

```diff
@@ -22,14 +21,18 @@ import com.atlas.sdk.data.InternalMessageHandler
 import com.atlas.sdk.repository.ConversationsRemoteRepository
 import com.atlas.sdk.repository.UserLocalRepository
 import com.atlas.sdk.repository.UserRemoteRepository
-import com.atlas.sdk.view.AtlasView
+import com.atlas.sdk.view.AtlasViewFragment
```

Comment:
I feel that since we replaced the "view" with "fragment", and "fragment" is common concept in android development we should reference it as "AtlasFragment", "getAtlasFragment" and so on